client/components/document/hooks/use-formatting-click.ts

```diff
@@ -0,0 +1,75 @@
+import type { RefObject } from "react";
```

Comment:
This is great but can we keep it in lib/?

Reply:
> This is great but can we keep it in lib/?

Off course